### ISSUE TITLE
/tmp/vagrant-cache permission error

### ISSUE THREAD

**Issue by mortona42** [forumone/puppet-forumone#50]
This leads to ruby and solr failing to install.

```
==> default: Error: Failed to set owner to '501': Operation not permitted - /tmp/vagrant-cache
==> default: Error: /Stage[main]/Forumone/File[/tmp/vagrant-cache]/owner: change from 502 to vagrant failed: Failed to set owner to '501': Operation not permitted - /tmp/vagrant-cache
```

**Comment by mortona42** [comment]
William says to uninstall vagrant-cachier.

**Comment by wwhurley** [comment]
Resolved by version 1.1.45 which removes setting owner of /tmp/vagrant-cache
